packages/ark/source/crypto/transactions/types/schemas.ts

```diff
@@ -30,8 +30,9 @@ export const transactionBaseSchema: Record<string, any> = {
 		signature: { $ref: "alphanumeric" },
```

Comment:
can we add a workflow specifically for when the schemas file is changed? We can define a path for github actions when it should run: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

that way we will have the workflow as a safeguard in case the files gets changed